Jenkinsfile

```
@@ -4,14 +4,14 @@ def lts = "2.249.3"
 def weekly = "2.266"
 def buildConfiguration = [
 /*
-  [ platform: "linux",   jdk: "8", jenkins: lts, javaLevel: "8" ],
-  [ platform: "windows", jdk: "8", jenkins: lts, javaLevel: "8" ],
-  [ platform: "linux",   jdk: "11", jenkins: lts, javaLevel: "8" ],
-  [ platform: "windows", jdk: "11", jenkins: lts, javaLevel: "8" ],
+  [ platform: "linux",   jdk: "8", jenkins: lts ],
+  [ platform: "windows", jdk: "8", jenkins: lts ],
+  [ platform: "linux",   jdk: "11", jenkins: lts ],
+  [ platform: "windows", jdk: "11", jenkins: lts ],
 */
   // Also build on recent weekly
-  [ platform: "linux",   jdk: "11", jenkins: weekly, javaLevel: "8" ],
-  [ platform: "windows", jdk: "11", jenkins: weekly, javaLevel: "8" ]
+  [ platform: "linux",   jdk: "11", jenkins: weekly ],
+  [ platform: "windows", jdk: "11", jenkins: weekly ]
```

Again, you should let the POM define the baseline unless you are specifically overriding it.

```suggestion
  [ platform: "linux",   jdk: "11" ],
  [ platform: "windows", jdk: "11" ]
```